docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,389 @@ The parent key for all of the following parameters is `rook_ceph`.
+
+* The data in parameter `images` is used to set the `image.repository`, `image.tag`, and `csi.cephcsi.image` Helm values
+* The value of `node_selector` is used to set Helm value `nodeSelector`
+* The value of `tolerations` is used to set Helm value `tolerations`
+
+See https://rook.io/docs/rook/v1.6/helm-operator.html#configuration[the Rook.io docs] for a full list of Helm values.
+
+== `toolbox`
+
+[horizontal]
+type:: dict
+default::
++
+[source,yaml]
+----
+enabled: true
+image: ${rook_ceph:images:rook}
+----
+
+The configuration for the Rook-Ceph toolbox deployment.
+This deployment provides an in-cluster shell to observe and administrate the Ceph cluster.
+
+== `cephClusterSpec`
+
+[horizontal]
+type:: dict
+default:: See https://github.com/projectsyn/component-rook-ceph/blob/master/class/defaults.yml[`class/defaults.yml` on Github]
+
+The default configuration for the `CephCluster` resource.
+The value of this parameter is used as field `.spec` of the resulting resource.
+
+Selected configurations of the Ceph cluster are inherited from other component parameters.
+If you overwrite those configurations in this parameter, the values provided in the "source" parameters won't have an effect.
+
+=== Inherited configurations
+
+* `cephVersion.image` is constructed from the data in parameter <<_images,`images`>>.
+* `placement.all.nodeAffinity` is built from parameter <<_node_selector,`node_selector`>>.
+The component constructs the following value for the configuration:
++
+[source,yaml]
+----
+requiredDuringSchedulingIgnoredDuringExecution:
+  nodeSelectorTerms:
+    - matchExpressions: <1>
+        - key: NODE_SELECTOR_KEY
+          operator: Exists
+        ...
+----
+<1> The component creates an entry in `matchExpressions` with `key` equal to the node selector key and `operator=Exists` for each key in parameter <<_node_selector,`node_selector`>>.
+
+* `placement.all.tolerations` is set to the value of parameter <<_tolerations,`tolerations`>>.
+* The component creates as single entry for `storage.storageClassDeviceSets` based on values given in parameter <<_ceph_cluster,`ceph_cluster`>>.
+Users are encouraged to use the parameter <<_ceph_cluster,`ceph_cluster`>> to configure the Ceph cluster's backing storage.
++
+The component expects that the provided storageclass for the backing storage supports `volumeMode=Block`.
+
+See https://rook.io/docs/rook/v1.6/ceph-cluster-crd.html#settings[the Rook.io `CephCluster` documentation] for a full list of configuration parameters.
+
+== Example configurations
+
+=== Configure and tune Ceph's backing storage
+
+To configure the component for an infrastructure which provides backing storage as storageclass `localblock-storage` and uses SSDs to provide local volumes the following config can be provided.
+
+[source,yaml]
+----
+parameters:
+  rook_ceph:
+    ceph_cluster:
+      block_storage_class: localblock-storage
+      tune_fast_device_class: true
+----
```

Comment:
This section could probably go into a How-to-tune page instead of the reference.

Reply:
Might be a good idea. However, I think that's something we can easily do in a follow-up story/PR

Reply:
I didn't think of a full-blown tuning page yet. Simply move this section and let it start small, grow organically. Should't take more than 5 mins I think.

But no hard feeling either.

Reply:
done in latest version